static/src/javascripts/projects/common/modules/discussion/comment-box.js

```diff
@@ -87,9 +87,7 @@ class CommentBox extends Component {
                 'Sorry, your comment was not published as you are no longer signed in. Please sign in and try again.',
             'READ-ONLY-MODE':
                 'Sorry your comment can not currently be published as commenting is undergoing maintenance but will be back shortly. Please try again in a moment.',
-            USERNAME_MISSING: `You must <a href="${config.get(
-                'page.mmaUrl'
-            )}/public-settings">set a username</a> before commenting. (<a href="/help/2020/feb/10/why-am-i-unable-to-post-a-comment">Learn more</a>).`,
+            USERNAME_MISSING: `You must <a href="${config.get('page.mmaUrl')}/public-settings">set a username</a> before commenting. (<a href="/help/2020/feb/10/why-am-i-unable-to-post-a-comment">Learn more</a>).`,
```

Comment:
Looks like this was accidently commited?

Reply:
I can only guess it was linted when I was pushing(?)

Reply:
I think you made this change the other day, it probably came over from your main branch